packages/backup-restore-operator/charts/README.md

```diff
@@ -0,0 +1,30 @@
+# Backup-Restore-Operator Chart
+
+The backup-restore-operator provides ability to back up and restore Kubernetes cluster metadata for user applications 
+
+### Components
+
+This chart installs the following clusterscoped CRDs  
+
+* **Backup**  
+  Creating an instance of the Backup CRD lets you perform a backup. A Backup CR must specify a ResourceSet, that contains a set of selectors to target certain resources/CRDs. (docs link TBD)
+* **Restore**  
+  Creating an instance of the Restore CRD lets you restore from a backup file. (docs link TBD)
+* **ResourceSet**  
+  ResourceSet CR should contain a set of selectors for all Kubernetes resources and CRDs that your application installs and manages, so that the operator can gather them during backups (docs link TBD)
+
+
+### User flow
+
+1. This chart can be configured to save all the backups at a particular location, such as s3/minio bucket, or in a Kubernetes Persistent Volume.
+2. The Backup CR must refer a ResouceSet, and the operator will collect and save resources that match its ResourceSet
+3. [This](linkTBD) is the ResourceSet that selects all resources installed by Rancher in the management cluster
+The chart will add this ResourceSet in the cluster during installation
+4. User can choose to encrypt resources before saving them in the backup (docs link TBD)
+5. For restore, user must specify the exact tarball filename that contains the backup data.
+
+
+### Installation
+```
+helm install backup-restore-operator ./ -n cattle-resources-system --create-namespace 
```

Comment:
shouldn't this be `helm install --create-namespace  -n cattle-resources-system backup-restore-operator rancher/stable`? Also, we should mention something about installing the CRD chart first here

Reply:
I'll add in rancher/stable. But there's no separate CRD chart, if user is using helm cli, the /crds folder will install them right?

Reply:
Hmm maybe we should split the Installation into two sections:

*Note: added a backslash before the code block to allow Github to render a code block in a code block*

```
#### Using `rancher/stable`

If installing this from the `rancher/stable` repo, use the following commands
\```bash # 
helm install --create-namespace -n cattle-resources-system backup-restore-operator-crd rancher/stable
helm install --create-namespace -n cattle-resources-system backup-restore-operator rancher/stable
\```bash

The CRDs used by this chart are packaged in a separate chart in the `rancher/stable` repo to allow users to keep CRDs installed onto a cluster even if the main package is deleted. You can create, update, or delete the CRDs packaged in that chart by using the appropriate `helm install`, `helm upgrade`, and `helm uninstall` commands on the CRD chart.

#### Installing this chart from this repo

If downloading this chart from this repo directly, use
\```bash
helm install --create-namespace -n cattle-resources-system backup-restore-operator <path-to-chart> 
\```.

This form of the chart will package the CRDs directly within the chart. Since this uses Helm 3's `crd/` directory, these CRDs will only be installed onto the cluster and not upgraded / removed on a `helm upgrade` or `helm uninstall`. To clean up the CRDs from the chart on uninstall, please manually execute the following commands:

\```bash
kubectl -n cattle-resources-system delete crd <crd-name>
... (Note: repeat three times for each of the CRDs installed by this chart)
\```
```